willowtest/specs/earthstar/es6_spec.ts

```diff
@@ -289,5 +327,17 @@ a04fc7...c37466...|91715a...f0eef3...|000000...  |000000...   |00`),
 				]),
 				
 			]),
+			
+			hsection("es6_friendly_paths", "Friendly paths", [
+				pinformative("While Willow's ", rs("Path"), " are defined as sequences of bytestrings, Earthstar defines a subset of these as human-readable ", def({
+					id: "es6_friendly_path",
+					singular: "friendly path",
+					plural: "friendly paths",
+				}, "friendly paths"), "."),
+				pinformative("A path may be considered ", r("es6_friendly_path", 'friendly'), " if every byte of its bytestrings belong to the set of ascii encodings of the following characters: ", code("-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz_"), ", that is, alphanumerics and ", code("-"), ", ", code("."), ", and ", code("_"), "."),
```

Comment:
```suggestion
				pinformative("A path is considered ", r("es6_friendly_path", 'friendly'), " if every byte of its bytestrings belong to the set of ascii encodings of the following characters: ", code("-.0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz_"), ", that is, alphanumerics and ", code("-"), ", ", code("."), ", and ", code("_"), "."),
```

Reply:
Okay, I was leaving some room for paths which unintentionally happen to fall within these constraints, but why muddy the water.